frontend/server/src/Validators.php

```diff
@@ -61,6 +61,19 @@ public static function validateStringNonEmpty(
         }
     }
 
+    /**
+     * Check whether parameter value is string and not empty
+     *
+     * @param mixed $parameter
+     * @psalm-assert string $parameter
+     */
+    public static function stringNonEmpty($parameter): bool {
+        if (!is_string($parameter) || empty($parameter)) {
+            return false;
+        }
+        return true;
```

Comment:
```suggestion
        return !empty($parameter);
```

en general si hay algo de la forma `if (...) { return true; } return false;` (o lo contrario), se puede reemplazar por `return ....;`

Reply:
Listo.